account_move_promissory_note/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     """,
     "author": "Solvos",
     "license": "AGPL-3",
-    "version": "13.0.1.1.1",
+    "version": "13.0.1.2.1",
```

Comment:
Wrong version:
```suggestion
    "version": "13.0.1.2.0",
```

See https://github.com/solvosci/slv-docker-odoo-prod/wiki/Odoo-SLV-contribution-brief-guidelines#41-in-general